lib/src/components/_stacks-inputs.less

```diff
@@ -409,7 +413,15 @@ fieldset {
     }
 
     .s-input-message {
-        color: shade(@yellow-600, 25%);
+        color: #7C6400;
```

Comment:
Do you think this could use `@yellow-800` instead of hex value? It's almost the same color.

And for `a`, one line below, maybe use `@yellow-900`?

Otherwise, LGTM.